doc/user_guide/eds.rst

```diff
@@ -543,50 +561,81 @@ The obtained composition is in atomic percent, by default. However, it can be tr
     Fe (Fe_Ka): Composition = 4.96 weight percent
     Pt (Pt_La): Composition = 95.04 weight percent
 
-or with :py:func:`~.misc.material.atomic_to_weight`. The reverse method is :py:func:`~.misc.material.weight_to_atomic`.
+or using :py:func:`~.misc.material.atomic_to_weight`:
 
 .. code-block:: python
 
     >>> # With atomic_percent from before
     >>> weight_percent = hs.material.atomic_to_weight(atomic_percent)
 
-The zeta-factor method needs both the 'beam_current' (in nA) and the acquisition or dwell time (referred to as 'real_time' in seconds) in order to provide an accurate quantification. Both of the these parameters can be assigned to the metadata of the spectrum using:
+The reverse method is :py:func:`~.misc.material.weight_to_atomic`.
+
+The zeta-factor method needs both the 'beam_current' (in nA) and the acquisition
+or dwell time (referred to as 'real_time' in seconds) in order to obtain an accurate
+quantification. Both of the these parameters can be assigned to the metadata using:
 
 ..code-block:: python
 
     >>> s.set_microscope_parameters(beam_current=0.5)
     >>> s.set_microscope_parameters(real_time=1.5)
 
-If these are not set the code will produce an error stating which parameter has been forgotten.
-The zeta-factor method will produce two sets of results. Index [0] is the composition maps for each element in atomic percent and index [1] is the mass-thickness map.
+If these parameters are not set, the code will produce an error.
+The zeta-factor method will produce two sets of results. Index [0] contains the
+composition maps for each element in atomic percent, and index [1] contains the mass-thickness map.
 
-The cross section method needs the 'beam_current', dwell time ('real_time') and probe area in order to provide accurate quantification. The 'beam_current' and 'real_time' can be set using :py:meth:`~._signals.eds_tem.EDSTEMSpectrum.set_microscope_parameters` as shown above. The 'probe_area' (in nm^2) can be defined in two different ways. Either, the probe diameter is narrower than the pixel width, which case the probe is being under-sampled and therefore an estimation of the probe area needs to be used. This can be added to the metadata by:
+The cross section method needs the 'beam_current', dwell time ('real_time') and probe area
+in order to obtain an accurate quantification. The 'beam_current' and 'real_time' can be set as shown above.
+The 'probe_area' (in nm^2) can be defined in two different ways.
+
+If the probe diameter is narrower than the pixel width, then the probe is being
+under-sampled and an estimation of the probe area needs to be used. This can be added
+to the metadata with:
 
 ..code-block: python
```

Comment:
Now that you're at it you could also fix the missing space here, it should be: `.. code-block:: python`